deephyper/skopt/optimizer/optimizer.py

```diff
@@ -586,13 +608,13 @@ def ask(self, n_points=None, strategy="cl_min"):
             opt_yi = self._filter_failures(opt.yi)
 
             if strategy == "cl_min":
-                y_lie = np.min(opt_yi) if opt_yi else 0.0  # CL-min lie
+                y_lie = np.min(opt_yi, axis=0) if opt_yi else 0.0  # CL-min lie
```

Comment:
document the behaviour of the liar strategy in the Multi-objective case somewhere (selecting the min/mean/max of each objective

Reply:
Where in the code would you suggest I document this?